operator/backupcontroller/executor.go

```diff
@@ -143,6 +151,35 @@ func (b *BackupExecutor) listAndFilterPVCs(ctx context.Context, annotation strin
 	return backupItems, nil
 }
 
+// listBackupPods lists all Pods in the given namespace and filters them for K8up specific usage.
```

Comment:
I thought about that too, however this is not so straight forward as I thought it would be.
The `ListPods` requires a `PodLister`  and that in turn uses `newk8sClient` which doesn't seem to work in the operator out of the box. Will probably require some refactoring here and there.

Reply:
Interesting that the client won't work in the operator out of the box. What errors did you get?

Reply:
Pushed the changes now. The error was caused, because I ran k8up locally in my IDE and is therefore missing the in-cluster configuration.